apps/admin_panel/assets/src/omisego/config.js

```diff
@@ -1,5 +1,5 @@
 export const ADMIN_PANEL_BASE_DIR = "/admin";
 
 export const ADMIN_API_BASE_URL = "/admin/api";
-export const ADMIN_API_KEY_ID = process.env.ADMIN_API_KEY_ID;
-export const ADMIN_API_KEY = process.env.ADMIN_API_KEY;
+export const ADMIN_API_KEY_ID = window.admin_api_conf.api_key_id;
```

Comment:
🥇 🥇 🥇 

However, JavaScript convention is to use `camelCase` for variable and `UNDERSCORE_NAME` only for constants, so this is probably better as `window.adminConfig.apiKeyId` or something.